src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -2572,6 +2587,7 @@
 				71F5418B248BEDBE006DB793 /* de */,
 				717D21EA248C072300D9717E /* en */,
 				EEDD6DF624A4885200BC30D0 /* tr */,
+				EE902FEF24B3015C00C8DEE9 /* ru */,
```

Comment:
This looks odd. The usual pattern of your changed seams to be one line with `/* ru */` followed by a line with `/* ar */`.
This structure those not have an  `/* ar */`.

Reply:
good catch